include/device.h

```diff
@@ -487,6 +487,46 @@ device_required_handles_get(const struct device *dev,
 	return rv;
 }
 
+/**
+ * @brief Get the set of handles for devicetree supports of this device.
+ *
+ * These are the device supports inferred from devicetree.
```

Comment:
same here, what do "device supports" and "devicetree supports" mean in this context?